File: src/main/java/eladkay/quaritum/common/item/ModItems.java
package eladkay.quaritum.common.item;

import eladkay.quaritum.common.Quaritum;
import eladkay.quaritum.common.item.chalk.ItemChalk;
import eladkay.quaritum.common.item.chalk.ItemChalkTempest;
import eladkay.quaritum.common.item.misc.*;
import eladkay.quaritum.common.item.soulstones.*;

public class ModItems {

    public static ItemDormantSoulstone dormant;
    public static ItemAwakenedSoulstone awakened;
    public static ItemPassionateSoulstone passionate;
    public static ItemWorldSoulstone passive;
    public static ItemOppressiveSoulstone oppressive;
    public static ItemAttunedSoulstone attuned;
    public static ItemDebug debug;
    public static ItemChalk chalk;
    public static ItemFertilizer fertilizer;
    public static ItemReagentAtlas altas;
    public static ItemRiftmakerPart riftmakerPart;
    public static ItemWorldBlade worldBlade;
    public static ItemPicture picture;
    public static ItemBook book;
    public static ItemTransducer transducer;
    public static ItemChalkTempest tempest;
    public static ItemHollower hollower;
    public static ItemPlacer placer;
    public static void init() {
        dormant = new ItemDormantSoulstone();
        awakened = new ItemAwakenedSoulstone();
        passionate = new ItemPassionateSoulstone();
        passive = new ItemWorldSoulstone();
        oppressive = new ItemOppressiveSoulstone();
        attuned = new ItemAttunedSoulstone();
        chalk = new ItemChalk();
        fertilizer = new ItemFertilizer();
        altas = new ItemReagentAtlas();
        riftmakerPart = new ItemRiftmakerPart();
        worldBlade = new ItemWorldBlade();
        picture = new ItemPicture();
        transducer = new ItemTransducer();
        tempest = new ItemChalkTempest();
        book = new ItemBook();
        if (Quaritum.isDevEnv) {
            debug = new ItemDebug();
            hollower = new ItemHollower();
            placer = new ItemPlacer();
        }


    }
}

File: src/main/java/eladkay/quaritum/common/item/ModItems.kt
package eladkay.quaritum.common.item

import eladkay.quaritum.common.Quaritum
import eladkay.quaritum.common.item.chalk.ItemChalk
import eladkay.quaritum.common.item.chalk.ItemChalkTempest
import eladkay.quaritum.common.item.misc.*
import eladkay.quaritum.common.item.soulstones.*

object ModItems {

    var dormant: ItemDormantSoulstone
    var awakened: ItemAwakenedSoulstone
    var passionate: ItemPassionateSoulstone
    var passive: ItemWorldSoulstone
    var oppressive: ItemOppressiveSoulstone
    var attuned: ItemAttunedSoulstone
    lateinit var debug: ItemDebug
    var chalk: ItemChalk
    var fertilizer: ItemFertilizer
    var altas: ItemReagentAtlas
    var riftmakerPart: ItemRiftmakerPart
    var worldBlade: ItemWorldBlade
    var picture: ItemPicture
    var book: ItemBook
    var transducer: ItemTransducer
    var tempest: ItemChalkTempest
    lateinit var hollower: ItemHollower
    lateinit var placer: ItemPlacer

    init {
        dormant = ItemDormantSoulstone()
        awakened = ItemAwakenedSoulstone()
        passionate = ItemPassionateSoulstone()
        passive = ItemWorldSoulstone()
        oppressive = ItemOppressiveSoulstone()
        attuned = ItemAttunedSoulstone()
        chalk = ItemChalk()
        fertilizer = ItemFertilizer()
        altas = ItemReagentAtlas()
        riftmakerPart = ItemRiftmakerPart()
        worldBlade = ItemWorldBlade()
        picture = ItemPicture()
        transducer = ItemTransducer()
        tempest = ItemChalkTempest()
        book = ItemBook()
        if (Quaritum.isDevEnv) {
            debug = ItemDebug()
            hollower = ItemHollower()
            placer = ItemPlacer()
        }


    }
}
